agir/donations/admin/filters.py

```diff
@@ -0,0 +1,55 @@
+from django.urls import reverse
+from django.utils.html import format_html
+
+from agir.groups.models import SupportGroup
+from agir.lib.admin.autocomplete_filter import AutocompleteRelatedModelFilter
+from agir.lib.admin.form_fields import AutocompleteSelectModel
+from agir.people.models import Person
+
+
+class MonthlyAllocationSubscriptionPersonFilter(AutocompleteRelatedModelFilter):
+    title = "Personne"
+    parameter_name = "subscription_person"
+
+    def get_widget_instance(self):
+        return AutocompleteSelectModel(
+            Person,
+            self.model_admin.admin_site,
+        )
+
+    def get_queryset_for_field(self):
+        return Person.objects.all()
+
+    def queryset(self, request, queryset):
+        if not self.value():
```

Comment:
Le `not` a été rajouté pour corriger quelque chose ?

Reply:
Non, c'était une erreur, j'ai poussé trop vite